cdap-ui/cypress/integration/fieldlevellineage.spec.ts

```diff
@@ -39,6 +39,16 @@ describe('Generating and navigating field level lineage for datasets', () => {
       cy.get('[data-cy="Succeeded"]', { timeout: 360000 }).should('contain', 'Succeeded');
     });
   });
+  before(() => {
+    // make sure cookies are set to see v2 UI
+    cy.getCookie('CDAP_enable_experiments').then((cookie) => {
+      if (cookie && cookie.value === 'on') {
+        // toggle experiments off
+        cy.visit('cdap/ns/default/experimentToggle');
+        cy.get('.switch-button').click();
```

Comment:
I think cypress have some method to set the cookie, you don't need to actually navigate to the enable page.